packages/web/src/components/list/SingleDropdownList.js

```diff
@@ -172,6 +184,7 @@ class SingleDropdownList extends Component {
 					field: props.dataField,
 					size: props.size,
 					order: getAggsOrder(props.sortBy),
+					...(this.props.showMissing ? { missing: this.props.missingLabel } : {}),
```

Comment:
Refrain from using `props` from the context wherever possible. Since we call these functions from the lifecycle methods like `componentWillReceiveProps` where the updated props are not yet available on `this.props`. Hence, you should use `props` object coming from the params.